packages/components/src/custom-select-control/index.js

```diff
@@ -146,8 +148,9 @@ export default function CustomSelectControl( {
 								),
 								style: item.style,
 							} ) }
+							aria-selected={ index === valueIndex }
 						>
-							{ item === selectedItem && (
+							{ item.key === value.key && (
```

Comment:
Same as the other two usages of `value`. We need to use the `selectedItem` returned from `useSelect`.

Reply:
Hrm, because we're no longer telling `useSelect` about value updates, its `selectedItem` won't reflect the most recent value if that value is change elsewhere (as in the case of the multiple font-size setting options), so instead I'm checking whether a value has been set, and selecting the first item on the list if not (which I think would be expected behaviour in the case of an uncontrolled component).

Reply:
Ok, I've pushed my changes now 😄

Reply:
If the uncontrolled component updates, we can't keep selecting the first item in the list. We need to let Downshift tell us which item is selected.

Reply:
Oh, good catch. It should be working now (tested with the storybook component).